src/components/Modal/Modal.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { closeModal } from './../../actions/index';
+
+import { ReactComponent as Close } from './../../assets/Close.svg';
+import './Modal.scss';
+
+const Modal = (props) => {
+  const dispatch = useDispatch();
+  const modalState = useSelector(state => state.modals);
+
+  if (modalState) {
+    return (
+      <div className='modal'>
+        <div className='modal--overlay' onClick={() => dispatch(closeModal())}></div>
+        <div className='modal--container'>
+          <Close className='modal--close' onClick={() => dispatch(closeModal())}/>
+          <div className='modal--content'>
```

Comment:
```suggestion
          <div className='modal--body'>
```

since a modal will sometimes include a form lets give it a better class name that is more dynamic, content means just text but that not always true for this modal